### PR TITLE
feat: add animation begin support on web

### PR DESCRIPTION
**Pull request by nicholasyan**
## Summary

This PR adds a prop to expose the underlying web SVG `animation` element's [`begin`](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/begin) attribute.

(My specific motivation is to then be able to use this field to stagger a series of loading component animations, but I can imagine broader use-cases as well given the other possible values of the `begin` attribute.)

## Related Issue #[issue number]

None

## Any Breaking Changes

None

## Checklist
- [x] Are all the test cases passing?
- [x] If any new feature has been added, then are the test cases updated/added?
- [x] Has the documentation been updated for the proposed change, if required?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -126,6 +126,7 @@ const MyLoader = () => (
...
 | **`uniqueKey?: string`** <br /> Defaults to random unique id     | React DOM only             | Use the same value of prop key, <br/>that will solve inconsistency on the SSR, see more [here](https://github.com/danilowoz/react-content-loader/issues/78).                                                                                                                                        |
 | **`beforeMask?: JSX.Element`** <br /> Defaults to null           | React DOM<br/>React Native | Define custom shapes before content, <br/>see more [here](https://github.com/danilowoz/react-content-loader/issues/266).                                                                                                                                                                            |
+| **`animateBegin?: string`** <br /> Defaults to `undefined`              | React DOM only             | Delay before the animation begins, identical to the [SVG animate element begin attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/begin)
```
Comment (nicholasyan) [inline README.md:129]: Happy to co-locate this prop with the `animate` property above, but it looks like so far props in this list seem to have been added chronologically.

File: docs/index.stories.tsx [hunk 2]
```diff
@@ -468,3 +468,23 @@ export const BeforeMask = () => {
...
+      <ContentLoader animateBegin="5s" />
+    </>
+  )
+}
+
+animateBegin.story = {
+  parameters: {
+    notes: `##\`animateBegin?: string\` 
+    
+    Defaults to \`0s\`. This attribute accepts the same values as the SVG animate element's [\`begin\` attribute](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/begin).`,
```
Comment (nicholasyan) [inline docs/index.stories.tsx:488]: Note default to `0s` is only implicit here; the actual default value lives with the `begin` attribute itself on the `animation` element.

**Comment by nicholasyan** [comment]
@danilowoz would love to get feedback on this — let me know if there's anything I can do to help it get merged :)

**Review by danilowoz — APPROVED** [review]
Hey, sorry for the late answer! This looks good to me. Any reason why `package-lock.json` got so many changes? I think we could ignore it for while

**Pull request merged**